Documentation meta-descriptions review N° 3 (#5547)
wazuh/wazuh-documentation: source/user-manual/reference/tools/manage-agents.rst

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::

>   :description: The purpose of the manage_agents program is to provide an easy to use interface to handle authentication keys for Wazuh agents. Learn more about it here.

Reviewer: ```suggestion
  :description: The manage_agents program is a command-line tool to handle authentication keys for Wazuh agents. Learn more about it here.
```
